web/src/Components/IssueCard.elm

```diff
@@ -22,119 +24,118 @@ issueCard model ballotId =
         ballot =
             getBallot ballotId model
 
+        vote =
+            getVoteFromBallot ballotId model
+
         ballotDone =
             ballot.finish < model.now
 
-        clickMsg =
-            if ballotDone then
-                Nav <| NTo <| ResultsR ballotId
-            else
-                Nav <| NTo <| VoteR ballotId
-
-        cardColor =
+        isPending =
+            ballot.state
```

Comment:
this is super unclear and takes a lot of mental energy to parse - should be changed to testing elements, like `List.member ballot.state [BallotPendingCreation, BalltoPendingEdits, BallotPendingDelegation] || vote.state == Vote.Pending`

That's just one example - other ways to do it but this isn't good.

Reply:
this comment is just to make the above part of the review